src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -58,19 +58,9 @@
 ;;; risky for our downstream apps, thus it was decided that it makes sense to
 ;;; keep these overrides.
 (def acceptable-ciphers
-  ["TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA"
-   "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256"
-   "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA"
-
+  ["TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256"
    "TLS_DHE_RSA_WITH_AES_256_CBC_SHA256"
-   "TLS_DHE_RSA_WITH_AES_256_CBC_SHA"
-   "TLS_DHE_RSA_WITH_AES_128_CBC_SHA256"
-   "TLS_DHE_RSA_WITH_AES_128_CBC_SHA"
-
-   "TLS_RSA_WITH_AES_256_CBC_SHA256"
-   "TLS_RSA_WITH_AES_256_CBC_SHA"
-   "TLS_RSA_WITH_AES_128_CBC_SHA256"
-   "TLS_RSA_WITH_AES_128_CBC_SHA"])
```

Comment:
This should probably add a bunch more suites

Reply:
I added all of the cipher suites enabled by default on my local box using Java 11 that Jetty didn't warn about in a tongue-in-cheek commit at 3074952. I figured that at least might start a conversation about how to figure out what cipher suites we should be using.

Reply:
We should compare this to a valid cipher suite list for FIPS compliance

Reply:
I've pushed up the list of enabled cipher suites on a RHEL7 fips enabled VM you got @robdaemon (7027e2c). I then removed any that Jetty also warned were weak (4caf517). I also diffed those against the cipher suites in the PR previously that I had gotten from Java in OSX Mojave. Here is that diff:
```
tk-jetty9 justin (newjetty %) ‖ wc -l osx
      25 osx

tk-jetty9 justin (newjetty %) ‖ wc -l fips
      33 fips

tk-jetty9 justin (newjetty %) ‖ diff osx fips
1,2c1
< TLS_AES_128_GCM_SHA256
< TLS_AES_256_GCM_SHA384
---
> SSL_CK_DES_192_EDE3_CBC_WITH_SHA
11c10,17
< TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256
---
> TLS_DH_DSS_WITH_AES_128_CBC_SHA256
> TLS_DH_DSS_WITH_AES_128_GCM_SHA256
> TLS_DH_DSS_WITH_AES_256_CBC_SHA256
> TLS_DH_DSS_WITH_AES_256_GCM_SHA384
> TLS_DH_RSA_WITH_AES_128_CBC_SHA256
> TLS_DH_RSA_WITH_AES_128_GCM_SHA256
> TLS_DH_RSA_WITH_AES_256_CBC_SHA256
> TLS_DH_RSA_WITH_AES_256_GCM_SHA384
21a28
> TLS_ECDH_ECDSA_WITH_AES_256_GCM_SHA384
25a33
> TLS_SRP_SHA_DSS_WITH_AES_256_CBC_SHA
```

In other words osx is a subset of the fips suites save for three that RHEL7 didn't have: `TLS_AES_128_GCM_SHA256`, `TLS_AES_256_GCM_SHA384`, and `TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256`

The first two look to be TLSv1.3. (see https://crypto.stackexchange.com/questions/62917/what-is-different-below-two-ciphersuites). The third looks older though potentially still supported? I assume we can leave off all three for the time being?

Reply:
If the current list of cipher suites looks sane, I'll go ahead and rebase those commits away and update the other commit messages and I think we'll be good to go.

Reply:
Looks sane to me :+1: